### PR TITLE
Create dataset for Liu et al. (2024).

### PR DESCRIPTION
**Pull request by YWAN446**
I created the dataset for Liu et al. (2024). For now, I only included SARS-CoV-2 results for stool samples and a small proportion of demographic and clinical information from the dataset. Let's discuss, during our group meeting, how to include PMMoV and mtDNA results and other attribute information.

### CODE REVIEW

File: data/Liu2024Longitudinal.yaml [hunk 1]
```diff
@@ -0,0 +1,1648 @@
...
+    limit_of_detection: unknown
+    unit: gc/dry gram
+    reference_event: confirmation date
+participants:
+- attributes:
+    age: 44.0
+    sex: female
+    vaccinated: true
+  measurements:
+  - - analyte: stool_SARSCoV2_N1
```
Comment (tillahoffmann) [inline data/Liu2024Longitudinal.yaml:49]: Looks like the tests may be failing because of the double `- -` here.

Comment (YWAN446) [reply data/Liu2024Longitudinal.yaml:49]: I see. Let me fix it.

File: data/Liu2024Longitudinal.yaml [hunk 2]
```diff
@@ -0,0 +1,1648 @@
...
+    limit_of_quantification: 1000
+    limit_of_detection: unknown
+    unit: gc/dry gram
+    reference_event: confirmation date
+  stool_PMMoV:
+    description: >
+      PMMoV genome copy concentration in stool samples. The concentration were quantified
+      in gene copies per dry weight of stool.
+    specimen: stool
+    biomarker: PMMoV
```
Comment (tillahoffmann) [inline data/Liu2024Longitudinal.yaml:28]: Are there specific targets for PMMoV and mtDNA?

File: data/Liu2024Longitudinal.yaml [hunk 3]
```diff
@@ -0,0 +1,1648 @@
...
+    time: 48
+    value: 248334585.0
+  - analyte: stool_mtDNA
+    time: 50
+    value: 341572028.0
+  - analyte: stool_mtDNA
+    time: 55
+    value: 425112858.4
+- attributes:
+    age: .nan
```
Comment (tillahoffmann) [inline data/Liu2024Longitudinal.yaml:1132]: Is this `nan` intentional because the age of the participant is not known?

**Review by tillahoffmann — COMMENTED** [review]
This looks great! Couple of small questions.

**Comment by YWAN446** [comment]
Hi @tillahoffmann , I fixed the .nan value for age and edited the schema to allow the "unknown" value for "age". I also heard back from my colleague that the deindentified data is fine to be open. Please review and merge it if there are no further issues.

**Comment by tillahoffmann** [comment]
This looks great! Final question: Are there specific gene targets we want to report for PMMoV and/or mtDNA?

File: data/Liu2024Longitudinal.yaml (hunk 2 above)
Comment (YWAN446) [reply data/Liu2024Longitudinal.yaml:28]: There are no specific targets for PMMoV and mtDNA in the preprint. I could ask my colleague about it though.

Comment (tillahoffmann) [reply data/Liu2024Longitudinal.yaml:28]: Maybe we can merge this as-is and follow up at a later point if we need it?

Comment (YWAN446) [reply data/Liu2024Longitudinal.yaml:28]: Sure, let's do it. I have emailed my colleague to ask for the information of gene targets for PMMoV and mtDNA.

**Review by tillahoffmann — APPROVED** [review]

**Pull request merged**